cmd/doctor.go

```diff
@@ -96,7 +96,7 @@ func runRecreateTable(ctx *cli.Context) error {
 	setting.Cfg.Section("log").Key("XORM").SetValue(",")
 
 	setting.NewXORMLogService(!ctx.Bool("debug"))
-	if err := db.InitEngine(); err != nil {
+	if err := db.InitEngine(context.Background()); err != nil {
```

Comment:
hmm... need to determine if graceful is started here - if it is this should be graceful.HammerContext() if not we should be installingSignals if possible.